### DISCUSSION TITLE
Flop_phoenix for autocomplete / typeahead search in a modal

### DISCUSSION THREAD

**Discussion by kuatroka** [woylie/flop_phoenix#386] [Q&A] [answered]
Would `flop_phoenix` be ok to create a search autocomplete component in a modal with navigation to a selected page. where the search is based on a SQLite db? I will have a table of about 40000 items split into 3 categories, and when use clicks on a search items in the dropdown below the search window, it should create a new path based on the item category and id. For example the search could show up "Andorra" as a category 'city' or "Andorra" as 'country'. Maybe somebody got an outline of a reference code I can adapt for my case? Thanks

I'm looking for something like this
![flop_pagination_search](https://github.com/user-attachments/assets/7dfd5ec1-5bdc-4cce-b792-545e70807f33)

or this
![flop_pagination_search2](https://github.com/user-attachments/assets/4f27ece9-a28f-4f86-82a6-5797718b1019)

**Comment by woylie** [comment, marked as answer]
This doesn't really use core Flop functionality. I don't think this has a place in the library. [Elixir Forum](https://elixirforum.com/) is probably a better place to ask for advice.
